pkg/client/unversioned/events.go

```diff
@@ -184,6 +185,30 @@ func (e *events) Delete(name string) error {
 		Error()
 }
 
+// DeleteCollections deletes a collection of objects.
+func (e *events) DeleteCollection(options *api.DeleteOptions, listOptions unversioned.ListOptions) error {
```

Comment:
Maybe I missed it but what prevents multiple overlapping deletes of the same collection?